changelog_unreleased/api/15127.md

```diff
@@ -0,0 +1,21 @@
+#### Add typing to support `readonly` array properties of AST Node (#15127 by @auvred)
+
+```tsx
+interface TestNode {
+  regularArray: string[];
+  readonlyArray: readonly string[];
+}
+
+declare const path: AstPath<TestNode>;
+
+// Works fine
+path.map(() => "", "regularArray");
+
+/** Before */
+path.map(() => "", "readonlyArray");
+//                  ^ Argument of type '"readonlyArray"' is not assignable to parameter of type '"regularArray"'
+
+/** After */
+// Now works fine too
+path.map(() => "", "readonlyArray");
+```
```

Comment:
We use `Prettier stable` and `Prettier main` instead of `before/after`. https://github.com/prettier/prettier/blob/main/changelog_unreleased/TEMPLATE.md